components/scream/src/mct_coupling/scream_cxx_f90_interface.cpp

```diff
@@ -111,7 +106,7 @@ void scream_create_atm_instance (const MPI_Fint f_comm, const int atm_id,
     auto& ad = c.create<AtmosphereDriver>();
 
     ad.set_comm(atm_comm);
-    ad.set_params(ad_params);
+    ad.set_params(scream_params);
```

Comment:
should we call this `eamxx_params`?

Reply:
I guess. There's so much scream->eamxx to do... We can of course change this, but we can also wait until we change _all_ occurrences.